TrueDialog.Client.Standard/Helpers/Utils.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Text;
+
+namespace TrueDialog.Helpers
+{
+    public static class Utils
+    {
+        internal static string SoftReadPhoneNumber(string phoneNumber)
+        {
+            var rval = RemoveSpecialCharacters(phoneNumber);
+
+            if (rval[0] == '1' && rval.Length == 11)
+                return rval;
+            else if (rval[0] != '1' && rval.Length == 10)
+                rval = "1" + rval;
+            else
+                throw new ArgumentException("Wrong phone number format. Please follow E.164 format (+1xxxxxxxxxx).");
+
+            return rval;
+        }
+
+        internal static string ReadPhoneNumber(string phoneNumber)
```

Comment:
This looks the same as the one above...  Am I missing something or do we have two of the same method?